frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -254,90 +245,31 @@ export default {
         this.$emit("request_file_change", direction, file);
       }
     },
-    save: async function(){
```

Comment:
`save` method was moved to `annotation_ui_factory` and renamed to `process_audio_save` to match `text_annotation_core`